types/array.from/array.from-tests.ts

```diff
@@ -0,0 +1,12 @@
+import ArrayFrom = require('array.from');
+
+/**
+ * The `expectType` function from https://www.npmjs.com/package/tsd,
+ * except instead of returning `void`, it returns `T`.
+ */
+declare function expectType<T>(t: T): T;
```

Comment:
I have to use this because the type needed in the `// $ExpectType` would otherwise be very long and complex and error‑prone.